File: CommonHttp/comom_http/src/main/java/com/mcl/custom_dialog/IBaseDialog.java
package com.mcl.custom_dialog;

/**
 * @author MengChuiLiu
 */
public interface IBaseDialog {
    /**
     * 显示对话框
     */
    void show();

    /**
     * 取消对话框
     */
    void dismiss();

    /**
     * 对象销毁
     */
    void onDestroy();
}

File: CommonHttp/common_http/src/main/java/com/mcl/custom_dialog/IBaseDialog.kt
package com.mcl.custom_dialog

import android.content.DialogInterface

/**
 * @author MengChuiLiu
 */
interface IBaseDialog {
    /**
     * 显示对话框
     */
    fun show()

    /**
     * 取消对话框
     */
    fun dismiss()

    /**
     * 对象销毁
     */
    fun onDestroy()

    /**
     * 点击返回键监听
     */
    fun setOnCancelListener(onCancelListener: DialogInterface.OnCancelListener)
}
